test/validator.js

```diff
@@ -17,6 +17,30 @@ describe('validator', () => {
         expect(matches.length).to.be.equal(1);
     });
 
+    it('Ignorable sentence', async () => {
+        if (client.hasEmptyApiKey())
+            return;
+        var matches = [];
+        await client.getMatches('There are some new test failures.', matches);
```

Comment:
What about: `Test failure can be ignored` (different case)?

Reply:
This example should also be ignored with these changes.